File: prueban5_reactjs/src/Models/PermisoInfoDto.ts
export interface PermisoInfoDto {
    id:               number;
    nombreEmpleado:   string;
    apellidoEmpleado: string;
    tipoPermiso:      TipoPermisoDto;
    fechaPermiso:     Date;
}

export interface TipoPermisoDto {
    id:          number;
    descripcion: string;
}

File: PruebaN5.Api/PruebaN5.DOMAIN/DTOs/PermisoInfoDto.cs
using PruebaN5.DOMAIN.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PruebaN5.DOMAIN.DTOs
{
    public record PermisoInfoDto
    {
        public int Id { get; set; }
        public string NombreEmpleado { get; set; } = null!;
        public string ApellidoEmpleado { get; set; } = null!;
        public TipoPermiso TipoPermiso { get; set; } = null!;
        public DateTime FechaPermiso { get; set; }
    }
}
